// function throwError() { throw new Error('this is an error') }
const assert = require('assert')


test( 'percobaan ajah', () => {

  expect({a: 1, b: 2}).toEqual({a: 1})

} )